cluster-autoscaler/cloudprovider/gce/reserved.go

```diff
@@ -16,275 +16,11 @@ limitations under the License.
 
 package gce
 
-import (
-	"fmt"
-	"math"
-	"strconv"
-	"strings"
+// Reserved calculates the OS reserved values.
+type Reserved interface {
+	// CalculateKernelReserved computes how much memory Linux kernel will reserve.
+	CalculateKernelReserved(physicalMemory int64, os OperatingSystem, osDistribution OperatingSystemDistribution, nodeVersion string) int64
```

Comment:
Current implementation doesn't use the nodeVersion parameter and the code calling it passes empty string. Should the interface (via docstring) specify some expectations here? Ex. by explicitly saying nodeVersion is optional and clarifying the expected behavior if not specified?

Reply:
Added a comment that it optional. 
>Ex. by explicitly saying nodeVersion is optional and clarifying the expected behavior if not specified?
behavior depends on implementation, so not sure what to say.